apps/checkout-app/pages/api/payment-provider-settings.ts

```diff
@@ -24,4 +24,4 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
     return res.status(500).json({ error });
   }
 }
-export default allowCors(requireAuthorization(handler));
+export default allowCors(requireAuthorization(handler, ["HANDLE_PAYMENTS"]));
```

Comment:
Looking at how permissions are hardcoded here, as I understand they'll be passed down to requireAuthorization -> isAuthorized ->  hasPermissions -> and then this condition `if (!permissions?.length)` will always return the same value, is that correct?